CHANGELOG.md

```diff
@@ -166,6 +166,7 @@ Versions are `MAJOR.PATCH`.
 - [#55823](https://github.com/saltstack/salt/pull/55823) - Fix issue with overly long names in the LGPO module - [@twangboy](https://github.com/twangboy)
 - [#55843](https://github.com/saltstack/salt/pull/55843) - Fixed `file.mkdir` to respect `test=True` - [@mchugh19](https://github.com/mchugh19)
 - [#55845](https://github.com/saltstack/salt/pull/55845) - Fixed logging to return multiprocessing queue if it's already set - [@s0undt3ch](https://github.com/s0undt3ch)
+- [#56325](https://github.com/saltstack/salt/pull/56325) - Fix hyperlinks to `salt.serializers` and other documentation issues - [@myii](https://github.com/myii)
```

Comment:
Oooh, I think you need to rebase - the version above is `## Unreleased (Neon)`. The latest master should have `3000.1`, though given our current schedule this will probably actually end out in `## Unreleased - 3001 (Magnesium)`, which I don't think I added yet 🙃

Reply:
https://github.com/saltstack/salt/commit/e3216db3e51737c534c9b2a0154242600eaad3c2 That comment was correct - I hadn't added the Magnesium release to the top of the changelog

Reply:
@waynew Done, apologies for the mix-up.

Reply:
@waynew I just noticed:

> ```
> ### Unreleased (3000.1)
> ```

Surely that should be:

```diff
-### Unreleased (3000.1)
+## Unreleased (3000.1)
```

Reply:
🤦‍♂ doh! My bad. Yes it should. We'll probably fix that with Sodium - the changelog is important, but I wouldn't say it's *critical* 😝